lib/config/DefaultSettings.ts

```diff
@@ -2,6 +2,7 @@ import type { PluginSettings } from "./PluginSettings";
 
 export const DEFAULT_SETTINGS: PluginSettings = {
   apiToken: null,
+  autoRefreshInterval: 0,
```

Comment:
Other than pre-fetching the workspace data, the api refresh isn't too heavy. Since this fixes a common problem with users, I would consider turning this one on by default actually. Perhaps at an interval of 60 seconds?

Reply:
Switched to 1 minute default (and now measured in units of minutes). It's a bit odd to have the refresh interval configurable but not the polling interval; perhaps might be good to also expose the polling interval as a setting, or if changing the interval isn't necessary, to make auto-refresh a simple boolean toggle.

Reply:
I agree with that, I don't like exposing settings to users that could have a one-size-fits-all default.

Since the refresh still fetches some workspace data, a default of 2.5 minutes (150 seconds) seems good to me?